tsp/configuration.py

```diff
@@ -71,11 +73,23 @@ def __init__(self, params):
         else:
             self.parameters = {}
 
+    def __repr__(self):
+        return 'Configuration(name={}, description={}, id={}, source_type_id={}, parameters={})'.format(
+            self.name, self.description, self.id, self.source_type_id, self.parameters)
+
+    def to_json(self):
+        return json.dumps(self, cls=ConfigurationEncoder, indent=4)
+
+
+class ConfigurationEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Configuration):
+            return {
+                ID_KEY: obj.id,
+                NAME_KEY: obj.name,
+                DESCTIPION_KEY: obj.description,
```

Comment:
Constant should be renamed to DESCRIPTION_KEY

Reply:
Done